### PR TITLE
Hexen: Add mouse inventory buttons

### PR DESCRIPTION
**Pull request by mikeday0**
Duplication of e9e76ed, but for Hexen. All credit to @kraflab. I copied and pasted the Heretic implementation and it seems to work perfectly.

### CODE REVIEW

File: src/hexen/g_game.c [hunk 1]
```diff
@@ -744,6 +744,59 @@ static void SetJoyButtons(unsigned int buttons_mask)
     }
 }
 
+static boolean InventoryMoveLeft()
```
Comment (turol) [inline src/hexen/g_game.c:747]: Add comment about what the return value means.

**Comment by turol** [comment]
I'd prefer this split into two commits: first add the helper functions, then the mouse buttons.

**Comment by mikeday0** [comment]
> I'd prefer this split into two commits: first add the helper functions, then the mouse buttons.

No problem. I assume it's okay to force push to do this?

**Comment by mikeday0** [comment]
How would you feel about me adding a mouse button for "use artifact"?

**Comment by fabiangreffrath** [comment]
> How would you feel about me adding a mouse button for "use artifact"?

Fine with me!

**Comment by mikeday0** [comment]
> > How would you feel about me adding a mouse button for "use artifact"?
> 
> Would you like to add it to this PR? Because else I'd like to merge this as is.

Feel free to go ahead with the merge, it's probably cleaner to do this with a separate PR.

**Pull request merged**